packages/test/test-end-to-end-tests/src/test/gc/gcContainerRuntimeCompat.spec.ts

```diff
@@ -44,7 +41,7 @@ describeFullCompat("GC summary compatibility tests", (getTestObjectProvider) =>
         []);
     const runtimeOptions: IContainerRuntimeOptions = {
         summaryOptions: { disableSummaries: true },
-        gcOptions: { gcAllowed: true },
+        gcOptions: { gcAllowed: true, writeDataAtRoot: true },
```

Comment:
Aren't you removing `writeDataAtRoot` in the files above?

Reply:
Yes, this is for compatibility. Older versions of runtime still expect this option to write the data at root.

Reply:
...or is this simulating an older version that would have had writeDataAtRoot defined?